packages/client/components/MeetingsDash.tsx

```diff
@@ -75,15 +81,18 @@ const MeetingsDash = (props: Props) => {
 
         return a.createdAt > b.createdAt ? -1 : 1
       })
-    const filteredMeetings = dashSearch
-      ? meetings.filter(({name}) => name && name.match(getSafeRegex(dashSearch, 'i')))
-      : meetings
+    const searchedMeetings = dashSearch
+      ? sortedMeetings.filter(({name}) => name && name.match(getSafeRegex(dashSearch, 'i')))
+      : sortedMeetings
+    const filteredMeetings = searchedMeetings.filter((node) =>
+      teamIds ? teamIds.includes(node.teamId) : true
+    )
```

Comment:
+1 Move the ternary outside the `filter`? I.e. `teamIds ? searchedMeetings.filter(...) : searchedMeetings`, similar to the ternary for `searchedMeetings`